eng/Versions.props

```diff
@@ -322,12 +322,12 @@
     <SwashbuckleAspNetCoreVersion>6.6.2</SwashbuckleAspNetCoreVersion>
     <VerifyXunitVersion>19.14.0</VerifyXunitVersion>
```

Comment:
Not a requirement for this PR but we can probably bump the version of the Verify.Xunit dependency that we use now.

Reply:
Yes I looked at that but given the PR is already quite large I decided not to add unrelated changes.